JSDemos/Demos/PivotGrid/OLAPDataSource/description.md

```diff
@@ -1 +1,20 @@
-The PivotGrid supports OLAP services (MS&nbsp;SQL Server Analysis Services). This allows you to&nbsp;deliver solutions that can easily analyze millions of&nbsp;data rows within your app. In&nbsp;this demo, the PivotGrid uses **XmlaStore** to&nbsp;connect the component to&nbsp;a&nbsp;sample remote OLAP service. The Field Chooser is&nbsp;configured to&nbsp;show the search box to&nbsp;speed up&nbsp;choosing the required field.
\ No newline at end of file
+The PivotGrid component supports OLAP services (Microsoft SQL Server Analysis Services). This demo shows how to use a remote OLAP cube as the PivotGrid's data source.
+
+## Configure the Store
+To connect your component to an OLAP storage, use [XmlaStore](/Documentation/ApiReference/Data_Layer/XmlaStore/). Specify the following properties to configure it:    
+
+- [url](/Documentation/ApiReference/Data_Layer/XmlaStore/Configuration/#url)        
+The OLAP server's URL.
+
+- [catalog](/Documentation/ApiReference/Data_Layer/XmlaStore/Configuration/#catalog)        
+The initial catalog that contains the OLAP cube.
```

Comment:
What do you mean by "initial"?

Reply:
I guess that we can remove this word. It really doesn't add anything to this sentence.

Reply:
OK.
```suggestion
The catalog that contains the OLAP cube.
```